README.rst

```diff
@@ -1,142 +1,68 @@
-******************
 Specutils
-******************
-An AstroPy affiliated package containing operations and tools on astronomical spectra.
+=========
 
-First Steps
-==============
-The `Spectrum1D`_ class is one of the core classes of the specutils package.
-You can import it like this:
+.. image:: http://img.shields.io/pypi/v/specutils.svg?text=version
+    :target: https://pypi.python.org/pypi/specutils/
+    :alt: Latest release
 
-.. code-block:: python
-  
-  from specutils import Spectrum1D
+.. image:: https://anaconda.org/astropy/specutils/badges/version.svg
+    :target: https://anaconda.org/astropy/specutils
 
-To instantiate it you can define a wave and a flux:
+.. image:: https://anaconda.org/astropy/specutils/badges/downloads.svg
+    :target: https://anaconda.org/astropy/specutils
 
-.. code-block:: python
-  
-  wave = np.arange(6000, 9000) * u.Angstrom
-  flux = np.random.random(3000) * u.Unit('W m-2 angstrom-1 sr-1')
+.. image:: http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat
+    :target: http://www.astropy.org/
 
-and then call the **from_array** method:
+.. raw:: html
 
-.. code-block:: python
+    <br />
 
-  spec1d = Spectrum1D.from_array(wave, flux)
-  spec1d.wavelength
-  <Quantity [ 6000., 6001., 6002.,...,  8997., 8998., 8999.] Angstrom>
-  spec1d.flux
-  <Quantity [ 0.75639906, 0.23677036, 0.08408417,...,  0.82740303, 0.38345114,
-              0.77815595] W / (Angstrom m2 sr)>
+Specutils is an `Astropy`_ affiliated package with the goal of providing a
+shared set of Python representations of astronomical spectra, and basic tools to
+operate on these spectra.  The effort is also meant to be a "hub", helping to
+unite the Python astronomical spectroscopy community around shared effort, much
+as `Astropy`_ is meant to for the wider astronomy Python ecosystem.
 
-Or you can read a Spectrum from a .fits file with the **read_fits** method:
+Note that Specutils is *not* intended as an all-in-one spectroscopic analysis or
+reduction tool.  While it provides some baseline analysis (following the Python
```

Comment:
maybe `basic` instead of `baseline` ("baseline" = "continuum" in radio parlance)